src/ui/Facet/ValueElementRenderer.ts

```diff
@@ -262,6 +262,9 @@ export class ValueElementRenderer {
     el.setAttribute('aria-label', this.ariaLabel);
     el.setAttribute('role', 'button');
     el.setAttribute('aria-pressed', this.ariaPressed);
+
+    const elExclude = this.excludeIcon;
```

Comment:
As per my comment, you can remove that part and the method at line #282 😉

Reply:
In the JIRA issue, it says the Exclude button should have aria-pressed="mixed".  But I agree the state of the Exclude button isn't affected by the state of the Select button. 

@btaillon Is `aria-pressed="mixed"` really what we want?

Reply:
Hmm that's not how I understood it, sorry. 😅 
The Exclude button always has the state pressed or not, either it excludes or unexcludes.
The Select button selects, deselects, and unexcludes if exclude is pressed.

Reply:
Woops, that's my bad.